swagger.json

```diff
@@ -4146,24 +4143,28 @@
                 },
                 "eldExempt": {
                     "type": "boolean",
-                    "description": "Flag indicating this driver is exempt from the Electronic Logging Mandate."
+                    "description": "Flag indicating this driver is exempt from the Electronic Logging Mandate.",
+                    "example": false
                 },
                 "eldExemptReason": {
                     "type": "string",
-                    "description": "Reason that this driver is exempt from the Electronic Logging Mandate (see eldExempt)."
+                    "description": "Reason that this driver is exempt from the Electronic Logging Mandate (see eldExempt).",
+                    "example": "Some reason."
                 },
                 "usBigDayExemptionEnabled": {
                     "type": "boolean",
-                    "description": "Flag indicating this driver may use Big Day exemptions in HOS logs."
+                    "description": "Flag indicating this driver may use Big Day exemptions in HOS logs.",
```

Comment:
Is it necessary to add something about "US" in the description? Just wondering since it's in the variable name.

Reply:
Jonte gave me this description :)